only_otters/scrapetools/hquery.py

```diff
@@ -0,0 +1,317 @@
+import yaml
+import requests
+from lxml import html as lhtml
+from functools import partial
+import re
+
+import scrapetools.util as util
+import scrapetools.autobrowser as autobrowser
+
+# import util
+# import autobrowser
+
+
+class HierarchicalXPathQuery:
+
+    @classmethod
+    def resolve_mode(cls, name, high=False):
```

Comment:
resolve this, resolve that ...
A better terminology (pipes, modes are also confusing terms) would be useful in this module.